src/telemetry/step_stats.rs

```diff
@@ -35,13 +35,13 @@ impl CsvExporter for Metrics {
         // then dump them to the provided Write interface
         // TODO: include role dimension. That requires rethinking `Metrics` implementation
         // because it does not allow such breakdown atm.
-        writeln!(w, "Step,Records Sent,Indexed PRSS,Sequential PRSS")?;
+        writeln!(w, "Step,Bytes sent,Indexed PRSS,Sequential PRSS")?;
```

Comment:
Why not both?

Reply:
I was thinking that number of records may no longer be relevant now, but I don't mind adding it back

Reply:
I think it’s really helpful for analyzing where we should focus on optimizations.